src/main/haskell/kore/src/Data/Function/Compose.hs

```diff
@@ -2,7 +2,7 @@
 Module      : Data.Function.Compose
 Description : Short-hand notation for function composition with many arguments
 Copyright   : (c) Runtime Verification, 2018
-License     : UIUC/NCSA
+License     : NCSA
```

Comment:
Why the license change?

Reply:
The license itself does not change, but `NCSA` is the proper [SPDX license identifier](https://spdx.org/licenses/) for the license we're using.